### PR TITLE
[juicebox] strategy

### PR DESCRIPTION
**Pull request by drgorillamd**
This strategy returns the voting power of token holders of a Juicebox powered project (identified by projectId and symbol)

Please review me carefully, I'm a new contributor to snapshot
Test results:
image

### CODE REVIEW

File: src/strategies/juicebox/index.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'drgorillamd';
+export const version = '1.0';
```
Comment (ChaituVR) [inline src/strategies/juicebox/index.ts:5]:
```suggestion
export const version = '0.1.0';
```

File: src/strategies/juicebox/index.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	outputs: [
+		{
+			name: '',
+			type: 'uint256',
+			internalType: 'uint256'
+		}
+	],
+	stateMutability: 'view'
+	}
+]
```
Comment (ChaituVR) [inline src/strategies/juicebox/index.ts:33]:
```suggestion
const abi = [ 'function balanceOf(address, uint256) view returns (uint256)' ]
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/juicebox/examples.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+[
+    {
+      "name": "Juicebox Projects",
+      "strategy": {
+        "name": "juicebox",
+        "params": {
+                "projectId": "0x01",
+                "symbol": "JBX"
+        }
```
Comment (ChaituVR) [inline src/strategies/juicebox/examples.json:9]:
```suggestion
        "name": "JBX",
        "params": {
                "projectId": "1",
                "symbol": "JBX"
        }
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by bonustrack — APPROVED** [review]

File: src/strategies/juicebox/index.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      'balanceOf',
+      [address, options.projectId]
+    ]),
+    { blockTag }
+  );
+
+  return Object.fromEntries(
+    response.map((value, i) => [
+      addresses[i],
+      parseFloat(formatUnits(value.toString(), '18'))
```
Comment (bonustrack) [inline src/strategies/juicebox/index.ts:36]: I think formatUnit expect a number as second param
```suggestion
      parseFloat(formatUnits(value.toString(), 18))
```
Is this always 18 decimals?

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**